app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -273,32 +280,35 @@ public void onFocusChange(View view, boolean focus) {
                                 adapter.setDropDownViewResource(R.layout.cpr_spinner_item);
                                 // Apply the adapter to the spinner.
                                 spinner_Team.setAdapter(adapter);
-//                                Globals.CurrentTeamToScout = spinner_Team.getSelectedItemPosition();
+                                Globals.CurrentTeamToScout = spinner_Team.getSelectedItemPosition();
                             }
```

Comment:
Can't use Position to get the TeamToScout.  This won't work.  Since we just changed the match number, you should set this to teamInMatch.get(0) instead (just default to the first team).